### PR TITLE
upgrade to 0.2.7

### PR DESCRIPTION
**Pull request by double-y**
* change do_dist.cols to calculate distances between columns
* add do_cmdscale
* change get_sentiment to word_to_sentiment
* add get_sentiment that uses sentimentr

Test
* devtools::test()

### CODE REVIEW

File: R/tools.R [hunk 1]
```diff
@@ -30,3 +30,100 @@ generate_examples <- function(function_name, args){
   print(knitr::kable(ret, format="markdown"))
   ret
 }
+
+#' Check package update from cran and requiredRpackage json
```
Comment (hideaki) [inline R/tools.R]: I think this function is about our internal process and should not be shared to public.

File: R/stats_wrapper.R [hunk 2]
```diff
@@ -175,3 +175,49 @@ do_svd.kv <- function(df,
...
+  loadNamespace("dplyr")
+  loadNamespace("tidyr")
+  pair1_col <- col_name(substitute(pair_name1))
+  pair2_col <- col_name(substitute(pair_name2))
+  value_col <- col_name(substitute(value))
+  grouped_col <- grouped_by(df)
+
+  name_col <- avoid_conflict(grouped_col, "name")
+
+  do_cmdscale_each <- function(df){
```
Comment (hideaki) [inline R/stats_wrapper.R]: I guess this is for group_by again, but some comment would help.

Comment (double-y) [inline R/stats_wrapper.R]: add a comment

File: R/tools.R (hunk 1 above)
Comment (double-y) [inline R/tools.R]: This will be moved to our private repository

**Pull request merged**